site/content/docs/manifest/pipeline.en.md

```diff
@@ -77,6 +77,27 @@ The URI that identifies the Docker image to use for this build project. As of no
 <span class="parent-field">build.</span><a id="build-buildspec" href="#build-buildspec" class="field">`buildspec`</a> <span class="type">String</span>
 Optional. The URI that identifies a buildspec to use for this build project. By default, Copilot will generate one for you, located at `copilot/pipelines/[your pipeline name]/buildspec.yml`.
 
+<span class="parent-field">build.</span><a id="additional-policy" href="#additional-policy" class="field">`additional_policy.`</a><a id="policy-document" href="#policy-document" class="field">`PolicyDocument`</a> <span class="type">Map</span>
```

Comment:
```suggestion
<span class="parent-field">build.</span><a id="build-additional-policy" href="#build-additional-policy" class="field">`additional_policy.`</a><a id="policy-document" href="#policy-document" class="field">`PolicyDocument`</a> <span class="type">Map</span>
```

Reply:
I added `PolicyDocument` as well in order for customers to understand that they need to specify `PolicyDocument` keyword followed by the document body. Wouldn't it be confusing for the customers to see only `additional_policy` keyword and then suddenly see `PolicyDocument` in the example code?